inc/blocks/ab-test/components/variant-panel.js

```diff
@@ -1,15 +1,42 @@
 import React from 'react';
+import styled from 'styled-components';
 
 import GoalPicker from '../../personalization/components/goal-picker';
 
 const {
 	PanelBody,
 	TextControl,
 	RangeControl,
+	Button,
+	Icon,
 } = wp.components;
-const { useDispatch } = wp.data;
+const {
+	useDispatch,
+	useSelect,
+} = wp.data;
 const { __ } = wp.i18n;
 
+const Preview = styled( Button ).attrs( {
```

Comment:
I think if this is copied from the component in the titles AB test code we could make into a shared component that can be reused. Not essential but good practice whenever possible.

Reply:
@roborourke Would the component file be located inside the `src/components` folder? Once it's working, do I then adjust the titles AB test code to use the same compnenet?

Reply:
Yeah I think that makes sense, and yes we'd want to use it in both places. At this point I'm thinking you could just do this in a follow up PR instead though. Lets try and get the core feature sorted today.